public/js/cita.json

```diff
@@ -1710,7 +1710,7 @@
     "title": "Leicester",
     "address": "3rd Floor\n60 Charles Street\nLEICESTER\nLE1 1FB",
     "phone": "0116 326 6326",
-    "hours": "Monday to Friday, 9am to 5pm",
+    "hours": "Monday to Friday, 9am to 15:30pm",
```

Comment:
Clear cache for [this page](https://www.pensionwise.gov.uk/locations/fdcde285-37fd-4858-b00a-4ee31b4488b1) :bikini: ?